### DISCUSSION TITLE
Not working migration guide link in release article

### DISCUSSION THREAD

**Discussion by Polostor** [rwf2/Rocket#2263] [Questions] [answered]
Hey guys,

I wanted to upgrade my version of Rocket from 0.4 to 0.5rc2 so I checked the official article and it linked me to the migration guide which was not working.

I don't know where to inform about it so I will just post it here... In the article https://rocket.rs/v0.5-rc/news/2022-05-09-version-0.5-rc.2/ **second paragraph** it is mentioned that:
> Please see the new [migration guide](https://rocket.rs/v0.5-rc/guide/upgrading-from-0.4) for complete ..

where the link is https://rocket.rs/v0.5-rc/guide/upgrading-from-0.4 but it  probably should be different as it returns 
![image](https://user-images.githubusercontent.com/5700145/178235642-851d0101-d9a5-4018-8d72-96ca0431ab8d.png)

**Maybe** the link could be probably changed to https://rocket.rs/v0.5-rc/guide/upgrading/ instead.

**EDIT:** Btw the same problem is in the [CHANGELOG.md](https://github.com/SergioBenitez/Rocket/blob/v0.5-rc/CHANGELOG.md) file

**Comment by SergioBenitez** [comment, marked as answer]
Filed as an issue at https://github.com/SergioBenitez/Rocket/issues/2267 and fixed in https://github.com/SergioBenitez/Rocket/commit/f64029f079882c1fe8e47d65d4a57ec260729bea.
